{"name": "To check the functionality of Complete Page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=94.0.4606.81)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00A0BDE3+2473443]\n\tOrdinal0 [0x009A6661+2057825]\n\tOrdinal0 [0x008B2438+1057848]\n\tOrdinal0 [0x0089BD36+965942]\n\tOrdinal0 [0x008F79F9+1341945]\n\tOrdinal0 [0x00904242+1393218]\n\tOrdinal0 [0x008F55AB+1332651]\n\tOrdinal0 [0x008D2104+1188100]\n\tOrdinal0 [0x008D2F59+1191769]\n\tGetHandleVerifier [0x00B92266+1549718]\n\tGetHandleVerifier [0x00C3D4A7+2250711]\n\tGetHandleVerifier [0x00A9718B+521403]\n\tGetHandleVerifier [0x00A96229+517465]\n\tOrdinal0 [0x009AB79D+2078621]\n\tOrdinal0 [0x009AFB58+2095960]\n\tOrdinal0 [0x009AFC92+2096274]\n\tOrdinal0 [0x009B9541+2135361]\n\tBaseThreadInitThunk [0x7617FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77E57A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77E57A4E+238]\n\n", "trace": "  File \"C:\\Python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"Feature\\steps\\StepDefination.py\", line 301, in step_impl\n    Creat_button = context.driver2.find_element_by_xpath(\"//a[text()='Create your quiz']\")\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 514, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 1238, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 418, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "When User Friend Click On Create Your Quiz Button", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=94.0.4606.81)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00A0BDE3+2473443]\n\tOrdinal0 [0x009A6661+2057825]\n\tOrdinal0 [0x008B2438+1057848]\n\tOrdinal0 [0x0089BD36+965942]\n\tOrdinal0 [0x008F79F9+1341945]\n\tOrdinal0 [0x00904242+1393218]\n\tOrdinal0 [0x008F55AB+1332651]\n\tOrdinal0 [0x008D2104+1188100]\n\tOrdinal0 [0x008D2F59+1191769]\n\tGetHandleVerifier [0x00B92266+1549718]\n\tGetHandleVerifier [0x00C3D4A7+2250711]\n\tGetHandleVerifier [0x00A9718B+521403]\n\tGetHandleVerifier [0x00A96229+517465]\n\tOrdinal0 [0x009AB79D+2078621]\n\tOrdinal0 [0x009AFB58+2095960]\n\tOrdinal0 [0x009AFC92+2096274]\n\tOrdinal0 [0x009B9541+2135361]\n\tBaseThreadInitThunk [0x7617FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77E57A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77E57A4E+238]\n\n", "trace": "  File \"C:\\Python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"Feature\\steps\\StepDefination.py\", line 301, in step_impl\n    Creat_button = context.driver2.find_element_by_xpath(\"//a[text()='Create your quiz']\")\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 514, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 1238, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 418, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1634815295377, "stop": 1634815295392}, {"name": "Then User Friend Become User And Navigate To Home Page", "status": "skipped", "start": 1634815295394, "stop": 1634815295394}], "start": 1634815295373, "stop": 1634815295394, "uuid": "63d05b56-d5c5-4aee-937b-ce9d14e3670d", "historyId": "d597327387c5dfc69ddbe90bf85a099a", "fullName": "Wowdare User Side End to End Functionality: To check the functionality of Complete Page", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Complete_page"}, {"name": "feature", "value": "Wowdare User Side End to End Functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}